2023-09-21 23:08:45.864 - DBeaver 23.2.0.202309041200 is starting
2023-09-21 23:08:45.864 - OS: Mac OS X 13.5.2 (aarch64)
2023-09-21 23:08:45.864 - Java version: 17.0.6 by Eclipse Adoptium (64bit)
2023-09-21 23:08:45.865 - Install path: '/Applications/DBeaver.app/Contents/Eclipse'
2023-09-21 23:08:45.865 - Instance path: 'file:/Users/hidongmin37/web_1900_hds/workspace/'
2023-09-21 23:08:45.865 - Memory available 64Mb/1024Mb
2023-09-21 23:08:45.869 - Create display
2023-09-21 23:08:45.952 - Initializing DBeaver 23.2.0
2023-09-21 23:08:45.953 - Host plugin: org.jkiss.dbeaver.ui.app.standalone 23.2.0.202309041200
2023-09-21 23:08:45.953 - Initialize desktop platform...
2023-09-21 23:08:46.007 - BounceCastle bundle found. Use JCE provider BC
> Start Core File Systems [org.eclipse.core.filesystem 1.10.0.v20230425-0513]
> Start Core Resource Management [org.eclipse.core.resources 3.19.0.v20230510-1240]
> Start 팀 지원 핵심 [org.eclipse.team.core 3.10.0.v20230317-0901]
> Start DBeaver UI Navigator [org.jkiss.dbeaver.ui.navigator 1.0.113.202309041200]
2023-09-21 23:08:46.271 - Initialize base platform...
2023-09-21 23:08:46.279 - Platform initialized (326ms)
> Start DBeaver UI Editors - Base [org.jkiss.dbeaver.ui.editors.base 1.0.113.202309041200]
2023-09-21 23:08:46.291 - Starting instance server at http://localhost:49611
2023-09-21 23:08:46.294 - Run workbench
> Start DBeaver UI [org.jkiss.dbeaver.ui 5.1.128.202309041200]
> Start Eclipse e4 작업영역 SWT [org.eclipse.e4.ui.workbench.swt 0.17.0.v20230501-1823]
> Start Eclipse Application Services [org.eclipse.e4.core.services 2.4.0.v20230429-1914]
> Start Eclipse UI Application Services [org.eclipse.e4.ui.services 1.6.0.v20230501-1823]
> Start Eclipse e4 Progress View [org.eclipse.e4.ui.progress 0.4.0.v20230501-1823]
> Start Eclipse Bindings Support [org.eclipse.e4.ui.bindings 0.14.0.v20230501-1823]
> Start Eclipse e4 core commands [org.eclipse.e4.core.commands 1.1.0.v20230501-1823]
> Start EMF XML/XMI Persistence [org.eclipse.emf.ecore.xmi 2.18.0.v20230211-1150]
> Start Eclipse e4 작업영역 기능 추가 [org.eclipse.e4.ui.workbench.addons.swt 1.5.0.v20230501-1823]
> Start Eclipse e4 작업영역 SWT 렌더러 [org.eclipse.e4.ui.workbench.renderers.swt 0.16.0.v20230525-0648]
> Start Eclipse Dependency Injection Extensions [org.eclipse.e4.core.di.extensions 0.18.0.v20230429-1914]
> Start Eclipse CSS SWT Support [org.eclipse.e4.ui.css.swt 0.15.0.v20230317-0802]
Warning: NLS unused message: BlockedJobsDialog_UserInterfaceTreeElement in: org.eclipse.ui.internal.progress.messages
> Start Eclipse CSS SWT Theme Support [org.eclipse.e4.ui.css.swt.theme 0.14.0.v20230501-1533]
2023-09-21 23:08:47.085 - Configure workbench window
> Start DBeaver SQL Model [org.jkiss.dbeaver.model.sql 1.0.103.202309041200]
2023-09-21 23:08:47.143 - Total database drivers: 105 (105)
> Start 도움말 시스템 코어 [org.eclipse.help 3.10.0.v20230317-0802]
2023-09-21 23:08:47.173 - Create actions
2023-09-21 23:08:47.175 - Disable Eclipse action set 'org.eclipse.ui.edit.text.actionSet.annotationNavigation'
2023-09-21 23:08:47.175 - Disable Eclipse action set 'org.eclipse.ui.WorkingSetActionSet'
2023-09-21 23:08:47.175 - Disable Eclipse action set 'org.eclipse.ui.actionSet.openFiles'
2023-09-21 23:08:47.309 - Initialize workbench window
2023-09-21 23:08:47.311 - Finish initialization
> Start 기본 문서 편집기 [org.eclipse.ui.editors 3.16.0.v20230329-1609]
> Start 텍스트 편집기 프레임워크 [org.eclipse.ui.workbench.texteditor 3.17.0.v20230316-1200]
> Start DBeaver UI Editors - Data [org.jkiss.dbeaver.ui.editors.data 1.0.113.202309041200]
Warning: NLS unused message: actions_navigator__objects in: org.jkiss.dbeaver.ui.editors.sql.internal.SQLEditorMessages
> Start DBeaver UI Editors - SQL [org.jkiss.dbeaver.ui.editors.sql 1.0.113.202309041200]
> Start DBeaver SQL Terminal [org.jkiss.dbeaver.ui.editors.sql.terminal 1.0.89.202309041200]
> Start 핵심 비교 지원 [org.eclipse.compare.core 3.8.0.v20230517-1315]
> Start 인터넷 연결 관리 사용자 인터페이스 [org.eclipse.ui.net 1.5.0.v20230317-0901]
> Start Internet Connection Management [org.eclipse.core.net 1.5.0.v20230317-0901]
> Start DBeaver Sample Database [org.jkiss.dbeaver.ext.sample.database 1.0.150.202309041200]
> Start DBeaver Tip of the day [org.jkiss.dbeaver.ext.ui.tipoftheday 1.0.119.202309041200]
> Start GEF Classic Draw2d [org.eclipse.draw2d 3.13.0.202306071707]
> Start GEF Classic GEF (MVC) [org.eclipse.gef 3.14.0.202306071707]
> Start DBeaver ERD UI [org.jkiss.dbeaver.erd.ui 3.0.73.202309041200]
> Start DBeaver ERD Model [org.jkiss.dbeaver.erd.model 1.0.73.202309041200]
> Start 콘솔 [org.eclipse.ui.console 3.12.0.v20230317-0802]
Warning: NLS unused message: InputStreamMonitor_label in: org.eclipse.debug.internal.core.DebugCoreMessages
Warning: NLS unused message: OutputStreamMonitor_label in: org.eclipse.debug.internal.core.DebugCoreMessages
> Start 디버그 핵심 [org.eclipse.debug.core 3.21.0.v20230508-1538]
2023-09-21 23:08:48.355 - Connect with 'localhost' (mysql8-18a7df49312-4eb0d6cf3dbcc505)
> Start DBeaver JDBC Model [org.jkiss.dbeaver.model.jdbc 1.0.6.202309041200]
> Start DBeaver MySQL Support [org.jkiss.dbeaver.ext.mysql 2.1.198.202309041200]
2023-09-21 23:08:48.375 - Execution context opened (localhost; Main; 0)
> Start DBeaver GIS Support [org.jkiss.dbeaver.data.gis 1.0.137.202309041200]
2023-09-21 23:08:48.481 - Execution context opened (localhost; Metadata; 1)
2023-09-21 23:08:48.518 - Connected (mysql8-18a7df49312-4eb0d6cf3dbcc505, MySQL Connector/J [mysql-connector-j-8.0.33 (Revision: 7d6b0800528b6b25c68b52dc10d6c1c8429c100c)])
2023-09-21 23:08:48.519 - Execution context opened (localhost; SQLEditor <User.sql>; 2)
> Start DBeaver Tasks - Native Tools UI [org.jkiss.dbeaver.tasks.native.ui 1.0.91.202309041200]
> Start DBeaver MySQL UI Support [org.jkiss.dbeaver.ext.mysql.ui 1.0.105.202309041200]
> Start DBeaver Tasks - General UI [org.jkiss.dbeaver.tasks.ui 1.0.91.202309041200]
2023-09-21 23:09:07.763 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:09:07.763 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:09:07.763 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:09:07.763 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:09:07.763 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:09:07.763 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:09:09.972 - Execution context opened (localhost; SQLEditor <Item.sql>; 3)
2023-09-21 23:09:14.792 - SQL Error [1046] [3D000]: No database selected
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1046] [3D000]: No database selected
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: No database selected
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
> Start DBeaver Tasks View [org.jkiss.dbeaver.tasks.ui.view 1.0.62.202309041200]
2023-09-21 23:09:56.413 - SQL Error [1824] [HY000]: Failed to open the referenced table 'sub_category2'
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1824] [HY000]: Failed to open the referenced table 'sub_category2'
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: Failed to open the referenced table 'sub_category2'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:10:03.458 - SQL Error [1060] [42S21]: Duplicate column name 'protected_payment'
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1060] [42S21]: Duplicate column name 'protected_payment'
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: Duplicate column name 'protected_payment'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:10:03.540 - SQL Error [1060] [42S21]: Duplicate column name 'protected_payment'
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1060] [42S21]: Duplicate column name 'protected_payment'
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: Duplicate column name 'protected_payment'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:10:21.135 - SQL Error [1060] [42S21]: Duplicate column name 'protected_payment'
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1060] [42S21]: Duplicate column name 'protected_payment'
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: Duplicate column name 'protected_payment'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
> Start DBeaver SQL Language Semantic Model [org.jkiss.dbeaver.model.lsm 1.0.8.202309041200]
2023-09-21 23:10:54.089 - Execution context opened (localhost; SQLEditor <ItemTag.sql>; 4)
2023-09-21 23:11:00.142 - SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'PRIMARY KEY (item_number, item_tag_number),
    FOREIGN KEY (item_number) REFERE' at line 1

Error position:
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'PRIMARY KEY (item_number, item_tag_number),
    FOREIGN KEY (item_number) REFERE' at line 1
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'PRIMARY KEY (item_number, item_tag_number),
    FOREIGN KEY (item_number) REFERE' at line 1
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:11:06.102 - SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'PRIMARY KEY (item_number, item_tag_number),
    FOREIGN KEY (item_number) REFERE' at line 1
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'PRIMARY KEY (item_number, item_tag_number),
    FOREIGN KEY (item_number) REFERE' at line 1
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'PRIMARY KEY (item_number, item_tag_number),
    FOREIGN KEY (item_number) REFERE' at line 1
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:11:07.812 - Execution context opened (localhost; SQLEditor <file.sql>; 5)
2023-09-21 23:11:13.037 - Execution context opened (localhost; SQLEditor <category.sql>; 6)
2023-09-21 23:11:38.373 - SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'PRIMARY KEY (item_number, item_tag_number),
    FOREIGN KEY (item_number) REFERE' at line 1
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'PRIMARY KEY (item_number, item_tag_number),
    FOREIGN KEY (item_number) REFERE' at line 1
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'PRIMARY KEY (item_number, item_tag_number),
    FOREIGN KEY (item_number) REFERE' at line 1
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:12:34.995 - SQL Error [1824] [HY000]: Failed to open the referenced table 'tbl_item_tag'
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1824] [HY000]: Failed to open the referenced table 'tbl_item_tag'
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: Failed to open the referenced table 'tbl_item_tag'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:13:30.604 - SQL Error [1822] [HY000]: Failed to add the foreign key constraint. Missing index for constraint 'fk_tag_relative_item_tag_number' in the referenced table 'tbl_item_tag'
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1822] [HY000]: Failed to add the foreign key constraint. Missing index for constraint 'fk_tag_relative_item_tag_number' in the referenced table 'tbl_item_tag'
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: Failed to add the foreign key constraint. Missing index for constraint 'fk_tag_relative_item_tag_number' in the referenced table 'tbl_item_tag'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:13:53.239 - SQL Error [1822] [HY000]: Failed to add the foreign key constraint. Missing index for constraint 'fk_tag_relative_item_tag_number' in the referenced table 'tbl_item_tag'
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1822] [HY000]: Failed to add the foreign key constraint. Missing index for constraint 'fk_tag_relative_item_tag_number' in the referenced table 'tbl_item_tag'
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: Failed to add the foreign key constraint. Missing index for constraint 'fk_tag_relative_item_tag_number' in the referenced table 'tbl_item_tag'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:14:01.201 - SQL Error [1833] [HY000]: Cannot change column 'item_tag_number': used in a foreign key constraint 'tbl_tag_relative_ibfk_2' of table 'coco_market.tbl_tag_relative'
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1833] [HY000]: Cannot change column 'item_tag_number': used in a foreign key constraint 'tbl_tag_relative_ibfk_2' of table 'coco_market.tbl_tag_relative'
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: Cannot change column 'item_tag_number': used in a foreign key constraint 'tbl_tag_relative_ibfk_2' of table 'coco_market.tbl_tag_relative'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
> Start ICU4J Base [com.ibm.icu.base 58.2.0.v20170418-1837]
2023-09-21 23:17:42.336 - SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CONSTRAINT fk_tag_relative_item_number REFERENCES tbl_item(item_number),
    FOR' at line 4
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CONSTRAINT fk_tag_relative_item_number REFERENCES tbl_item(item_number),
    FOR' at line 4
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CONSTRAINT fk_tag_relative_item_number REFERENCES tbl_item(item_number),
    FOR' at line 4
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:17:45.417 - SQL Error [1050] [42S01]: Table 'tbl_item_tag' already exists
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1050] [42S01]: Table 'tbl_item_tag' already exists
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: Table 'tbl_item_tag' already exists
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:17:47.145 - SQL Error [1822] [HY000]: Failed to add the foreign key constraint. Missing index for constraint 'fk_tag_relative_item_tag_number' in the referenced table 'tbl_item_tag'
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1822] [HY000]: Failed to add the foreign key constraint. Missing index for constraint 'fk_tag_relative_item_tag_number' in the referenced table 'tbl_item_tag'
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: Failed to add the foreign key constraint. Missing index for constraint 'fk_tag_relative_item_tag_number' in the referenced table 'tbl_item_tag'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:17:57.025 - SQL Error [1091] [42000]: Can't DROP 'fk_tag_relative_item_tag_number'; check that column/key exists
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1091] [42000]: Can't DROP 'fk_tag_relative_item_tag_number'; check that column/key exists
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: Can't DROP 'fk_tag_relative_item_tag_number'; check that column/key exists
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:17:58.552 - SQL Error [1833] [HY000]: Cannot change column 'item_tag_number': used in a foreign key constraint 'tbl_tag_relative_ibfk_2' of table 'coco_market.tbl_tag_relative'
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1833] [HY000]: Cannot change column 'item_tag_number': used in a foreign key constraint 'tbl_tag_relative_ibfk_2' of table 'coco_market.tbl_tag_relative'
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: Cannot change column 'item_tag_number': used in a foreign key constraint 'tbl_tag_relative_ibfk_2' of table 'coco_market.tbl_tag_relative'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:18:28.773 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:18:28.774 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:18:28.774 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:18:28.774 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:18:28.774 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:18:28.774 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:18:37.372 - SQL Error [1824] [HY000]: Failed to open the referenced table 'sub_category2'
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1824] [HY000]: Failed to open the referenced table 'sub_category2'
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: Failed to open the referenced table 'sub_category2'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:18:46.434 - Node 'coco_market' not found in parent node 'database://mysql8-18a7df49312-4eb0d6cf3dbcc505/org.jkiss.dbeaver.ext.mysql.model.MySQLCatalog'.
Allowed children: [animal [localhost], customer [localhost], jsp [localhost], mvc_member [localhost], mybatis [localhost], sys [localhost], test [localhost], todo [localhost]]
2023-09-21 23:18:46.434 - Node 'org.jkiss.dbeaver.ext.mysql.model.MySQLCatalog' not found in parent node 'database://mysql8-18a7df49312-4eb0d6cf3dbcc505'.
Allowed children: [Databases, Users, Administer, System Info]
2023-09-21 23:19:22.632 - SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CONSTRAINT fk_tag_relative_item_number REFERENCES tbl_item(item_number),
    FOR' at line 4
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CONSTRAINT fk_tag_relative_item_number REFERENCES tbl_item(item_number),
    FOR' at line 4
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CONSTRAINT fk_tag_relative_item_number REFERENCES tbl_item(item_number),
    FOR' at line 4
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:19:42.848 - SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CONSTRAINT fk_tag_relative_item_number REFERENCES tbl_item(item_number),
    FOR' at line 4

Error position: line: 3
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CONSTRAINT fk_tag_relative_item_number REFERENCES tbl_item(item_number),
    FOR' at line 4
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CONSTRAINT fk_tag_relative_item_number REFERENCES tbl_item(item_number),
    FOR' at line 4
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:19:49.256 - SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CONSTRAINT fk_tag_relative_item_number REFERENCES tbl_item(item_number),
    FOR' at line 4

Error position: line: 3
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CONSTRAINT fk_tag_relative_item_number REFERENCES tbl_item(item_number),
    FOR' at line 4
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CONSTRAINT fk_tag_relative_item_number REFERENCES tbl_item(item_number),
    FOR' at line 4
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:20:45.401 - SQL Error [1822] [HY000]: Failed to add the foreign key constraint. Missing index for constraint 'fk_tag_relative_item_tag_number' in the referenced table 'tbl_item_tag'
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1822] [HY000]: Failed to add the foreign key constraint. Missing index for constraint 'fk_tag_relative_item_tag_number' in the referenced table 'tbl_item_tag'
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: Failed to add the foreign key constraint. Missing index for constraint 'fk_tag_relative_item_tag_number' in the referenced table 'tbl_item_tag'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:22:46.327 - SQL Error [1091] [42000]: Can't DROP 'fk_tag_relative_item_tag_number'; check that column/key exists
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1091] [42000]: Can't DROP 'fk_tag_relative_item_tag_number'; check that column/key exists
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: Can't DROP 'fk_tag_relative_item_tag_number'; check that column/key exists
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:22:48.142 - SQL Error [1833] [HY000]: Cannot change column 'item_tag_number': used in a foreign key constraint 'tbl_tag_relative_ibfk_2' of table 'coco_market.tbl_tag_relative'
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1833] [HY000]: Cannot change column 'item_tag_number': used in a foreign key constraint 'tbl_tag_relative_ibfk_2' of table 'coco_market.tbl_tag_relative'
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: Cannot change column 'item_tag_number': used in a foreign key constraint 'tbl_tag_relative_ibfk_2' of table 'coco_market.tbl_tag_relative'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:24:26.427 - SQL Error [1049] [42000]: Unknown database 'coco_market'
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1049] [42000]: Unknown database 'coco_market'
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: Unknown database 'coco_market'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:26:58.843 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:26:58.843 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:26:58.843 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:26:58.844 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:26:58.844 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:26:58.844 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:27:08.904 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:27:08.904 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:27:08.904 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:27:08.904 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:27:08.905 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:27:08.905 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:27:08.905 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:27:08.905 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:27:08.905 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:27:08.905 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:27:08.905 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:27:08.905 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:27:08.905 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:27:23.978 - Table 'coco_market.tbl_item_tag' not found in metadata catalog
2023-09-21 23:27:23.978 - Table 'coco_market.tbl_item_tag' not found in metadata catalog
2023-09-21 23:27:30.764 - Table 'coco_market.tbl_item_tag' not found in metadata catalog
2023-09-21 23:27:30.764 - Table 'coco_market.tbl_item_tag' not found in metadata catalog
2023-09-21 23:27:31.671 - Table 'coco_market.tbl_tag_relative' not found in metadata catalog
2023-09-21 23:27:31.671 - Table 'coco_market.tbl_tag_relative' not found in metadata catalog
2023-09-21 23:27:38.090 - Table 'coco_market.tbl_file' not found in metadata catalog
2023-09-21 23:27:38.090 - Table 'coco_market.tbl_file' not found in metadata catalog
2023-09-21 23:27:38.090 - Table 'coco_market.tbl_file' not found in metadata catalog
2023-09-21 23:27:42.366 - Table 'coco_market.sub_category1' not found in metadata catalog
2023-09-21 23:27:42.367 - Table 'coco_market.sub_category1' not found in metadata catalog
2023-09-21 23:27:42.367 - Table 'coco_market.sub_category1' not found in metadata catalog
2023-09-21 23:27:43.521 - Table 'coco_market.sub_category2' not found in metadata catalog
2023-09-21 23:27:43.521 - Table 'coco_market.sub_category2' not found in metadata catalog
2023-09-21 23:27:43.521 - Table 'coco_market.sub_category2' not found in metadata catalog
2023-09-21 23:27:44.867 - Table 'coco_market.primary_category' not found in metadata catalog
2023-09-21 23:27:44.867 - Table 'coco_market.primary_category' not found in metadata catalog
2023-09-21 23:28:48.832 - Table 'coco_market.sub_category1' not found in metadata catalog
2023-09-21 23:28:48.832 - Table 'coco_market.sub_category1' not found in metadata catalog
2023-09-21 23:28:48.832 - Table 'coco_market.sub_category1' not found in metadata catalog
2023-09-21 23:28:50.252 - Table 'coco_market.sub_category2' not found in metadata catalog
2023-09-21 23:28:50.252 - Table 'coco_market.sub_category2' not found in metadata catalog
2023-09-21 23:28:50.252 - Table 'coco_market.sub_category2' not found in metadata catalog
2023-09-21 23:30:45.527 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:30:45.527 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:30:45.527 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:30:45.527 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:30:45.527 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:30:45.527 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:31:44.460 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:31:44.461 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:31:44.461 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:31:44.461 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:31:44.461 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:31:44.461 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-21 23:37:28.565 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:37:28.566 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:37:28.566 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:37:28.566 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:37:28.566 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:37:28.566 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:37:28.566 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:37:28.566 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:37:28.566 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:37:28.566 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:37:28.566 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:37:28.566 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:37:28.566 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-21 23:37:33.559 - Table 'coco_market.tbl_tag_relative' not found in metadata catalog
2023-09-21 23:37:33.559 - Table 'coco_market.tbl_tag_relative' not found in metadata catalog
2023-09-21 23:37:34.844 - Table 'coco_market.tbl_item_tag' not found in metadata catalog
2023-09-21 23:37:34.844 - Table 'coco_market.tbl_item_tag' not found in metadata catalog
2023-09-21 23:37:39.142 - Table 'coco_market.tbl_file' not found in metadata catalog
2023-09-21 23:37:39.143 - Table 'coco_market.tbl_file' not found in metadata catalog
2023-09-21 23:37:39.143 - Table 'coco_market.tbl_file' not found in metadata catalog
2023-09-21 23:51:03.399 - SQL Error [1824] [HY000]: Failed to open the referenced table 'sub_category2'
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1824] [HY000]: Failed to open the referenced table 'sub_category2'
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: Failed to open the referenced table 'sub_category2'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:51:24.920 - SQL Error [1824] [HY000]: Failed to open the referenced table 'tbl_item'
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1824] [HY000]: Failed to open the referenced table 'tbl_item'
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: Failed to open the referenced table 'tbl_item'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-21 23:51:37.487 - Table 'coco_market.tbl_file' not found in metadata catalog
2023-09-21 23:51:37.488 - Table 'coco_market.tbl_file' not found in metadata catalog
2023-09-21 23:51:37.489 - Table 'coco_market.tbl_file' not found in metadata catalog
2023-09-22 00:04:14.832 - Table 'coco_market.tbl_file' not found in metadata catalog
2023-09-22 00:04:14.833 - Table 'coco_market.tbl_file' not found in metadata catalog
2023-09-22 00:04:14.833 - Table 'coco_market.tbl_file' not found in metadata catalog
2023-09-22 00:06:07.762 - SQL Error [1824] [HY000]: Failed to open the referenced table 'tbl_item'
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1824] [HY000]: Failed to open the referenced table 'tbl_item'
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: Failed to open the referenced table 'tbl_item'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-22 00:06:13.217 - SQL Error [1050] [42S01]: Table 'tbl_item' already exists
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1050] [42S01]: Table 'tbl_item' already exists
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:582)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:491)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:498)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:934)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3940)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5142)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: Table 'tbl_item' already exists
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-09-22 00:07:58.062 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:07:58.063 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:07:58.063 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:07:58.063 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:07:58.063 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:07:58.063 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:33.154 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:33.154 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:33.154 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:33.154 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:33.154 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:33.154 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.095 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.095 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.095 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.095 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.095 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.095 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.308 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.308 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.308 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.308 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.308 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.308 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.448 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.448 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.448 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.448 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.448 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.448 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.596 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.596 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.596 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.596 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.596 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.596 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.715 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.715 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.715 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.715 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.715 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:08:34.715 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:33.824 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:33.825 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:33.825 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:33.825 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:33.825 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:33.825 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:34.924 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:34.924 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:34.924 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:34.924 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:34.924 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:34.924 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.116 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.116 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.116 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.116 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.116 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.116 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.265 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.265 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.265 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.265 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.265 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.265 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.407 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.407 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.407 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.407 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.407 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:13:35.407 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:14:56.305 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:14:56.305 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:14:56.305 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:14:56.305 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:14:56.305 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:14:56.305 - Table 'coco_market.tbl_user' not found in metadata catalog
2023-09-22 00:51:48.422 - Table 'coco_market.tbl_file' not found in metadata catalog
2023-09-22 00:51:48.423 - Table 'coco_market.tbl_file' not found in metadata catalog
2023-09-22 00:51:48.423 - Table 'coco_market.tbl_file' not found in metadata catalog
2023-09-22 00:53:24.458 - Table 'coco_market.tbl_file' not found in metadata catalog
2023-09-22 01:05:23.538 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-22 01:05:23.539 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-22 01:05:23.539 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-22 01:05:23.539 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-22 01:05:23.539 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-22 01:05:23.539 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-22 01:05:23.539 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-22 01:05:23.539 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-22 01:05:23.539 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-22 01:05:23.539 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-22 01:05:23.539 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-22 01:05:23.539 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-22 01:05:23.539 - Table 'coco_market.tbl_item' not found in metadata catalog
2023-09-22 01:15:24.277 - Execution context closed (localhost, 2)
2023-09-22 01:15:24.306 - Execution context closed (localhost, 4)
2023-09-22 01:15:24.337 - Execution context closed (localhost, 5)
2023-09-22 01:15:24.349 - Execution context closed (localhost, 6)
2023-09-22 01:15:24.400 - Execution context closed (localhost, 3)
2023-09-22 01:15:24.457 - DBeaver is stopping
2023-09-22 01:15:24.457 - Stop instance server
2023-09-22 01:15:25.458 - DBeaver shutdown completed
